Makefile

```diff
@@ -0,0 +1,63 @@
+SHELL = /bin/sh
+.DEFAULT_GOAL := help
+
+export IMAGE_PYTORCH=jupyter-ml-pytorch
+export IMAGE_TENSORFLOW=jupyter-ml-tensorflow
+export TAG_PYTORCH=2.0.0
```

Comment:
Can the bumpversion container handle two TAGs with the same version?

Reply:
yes, the trick was adding `"` to the integration version